DocumentedExamples/NearestNeighbourDistance.ipynb

```diff
@@ -0,0 +1,1728 @@
+{
```

Comment:
<ul><li>Let's remove the "Contributors"? Why do we need that?</li><li>Why "pixel" in grid? Perhaps "point in grid"?</li><li>What does "nearest neighbor" refer to?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
I applaud the credit :) thanks ❤️

But other examples don't have it... So it's confusing to somebody who might browse the examples later on?
Also it encourages the notion of users talking about an example and say "Navid's example about vorticity" as if I own the example or something. It's the community's example...

Anyway, these are personal thoughts and perhaps I shouldn't impose them or I should have explained my thinking behind the review comment.

Reply:
`xt_ocean, yt_ocean` are the cell's centers, yeah

Reply:
I'll have a closer look now at your PR :)